aopy/analysis/tuning.py

```diff
@@ -95,6 +95,69 @@ def get_mean_fr_per_condition(data, condition_labels, return_significance=False)
     else:
         return np.array(means_d).T, np.array(stds_d).T
     
+def convert_target_to_direction(target_locations):
+    '''
```

Comment:
Can you add an optional argument to define where the beginning target is? I think that would make it more generalizable to other tasks, like a random target task

Reply:
Sorry, I don't think I was clear. The zero axis and direction are definitely important and I'm glad ty, but so is the position of the start target in x,y coordinates. Right now it seems like the start target is always assumed to be at the center of the screen.

For example, if the target is at (0,5) (directly up from the center of the screen ) the angle will be 270 if the start target is at the center of the screen. If the start target is at (5,0) (directly right from the center of the screen) the angle will be 235.

Reply:
could we call that the "origin"? default (0,0)

Reply:
yup!

Reply:
what should be the angle if the target is at the origin? np.nan?

Reply:
yeah probably, and then in the docstring say that nans will occur if the target is at the origin input by the user. (Maybe a warning as well?)